skbio/stats/tests/test_composition.py

```diff
@@ -550,7 +550,121 @@ def test_ancom_basic_counts(self):
                             'reject': np.array([True, True, False, False,
                                                 False, False, False],
                                                dtype=bool)})
-        assert_data_frame_almost_equal(result, exp)
+        assert_data_frame_almost_equal(result[0], exp)
+
+    def test_ancom_percentiles(self):
+        table = pd.DataFrame([[12, 11],
+                              [9, 11],
+                              [1, 11],
+                              [22, 100],
+                              [20, 53],
+                              [23, 1]],
+                             index=['s1', 's2', 's3', 's4', 's5', 's6'],
+                             columns=['b1', 'b2'])
+        grouping = pd.Series(['a', 'a', 'a', 'b', 'b', 'b'],
+                             index=['s1', 's2', 's3', 's4', 's5', 's6'])
+        result = ancom(table, grouping)[1]
+        self.assertEqual(result.shape, (2, 10))
+        self.assertAlmostEqual(result[(0.0, 'a')]['b1'], 1.0)
+        self.assertAlmostEqual(result[(0.0, 'b')]['b1'], 20.0)
+        self.assertAlmostEqual(result[(25.0, 'a')]['b1'], 5.0)
+        self.assertAlmostEqual(result[(25.0, 'b')]['b1'], 21.0)
+        self.assertAlmostEqual(result[(50.0, 'a')]['b1'], 9.0)
+        self.assertAlmostEqual(result[(50.0, 'b')]['b1'], 22.0)
+        self.assertAlmostEqual(result[(75.0, 'a')]['b1'], 10.5)
+        self.assertAlmostEqual(result[(75.0, 'b')]['b1'], 22.5)
+        self.assertAlmostEqual(result[(100.0, 'a')]['b1'], 12.0)
+        self.assertAlmostEqual(result[(100.0, 'b')]['b1'], 23.0)
+
+        self.assertAlmostEqual(result[(0.0, 'a')]['b2'], 11.0)
+        self.assertAlmostEqual(result[(0.0, 'b')]['b2'], 1.0)
+        self.assertAlmostEqual(result[(25.0, 'a')]['b2'], 11.0)
+        self.assertAlmostEqual(result[(25.0, 'b')]['b2'], 27.0)
+        self.assertAlmostEqual(result[(50.0, 'a')]['b2'], 11.0)
+        self.assertAlmostEqual(result[(50.0, 'b')]['b2'], 53.0)
+        self.assertAlmostEqual(result[(75.0, 'a')]['b2'], 11.0)
+        self.assertAlmostEqual(result[(75.0, 'b')]['b2'], 76.5)
+        self.assertAlmostEqual(result[(100.0, 'a')]['b2'], 11.0)
+        self.assertAlmostEqual(result[(100.0, 'b')]['b2'], 100.0)
```

Comment:
Since we are on a tight deadline here, this is more of a future note.

We probably should be using assert_dataframe_equals here.  I'm not sure if it can handle MultiIndex objects at the moment.